### PR TITLE
MS-SQL binding support

### PR DESCRIPTION
**Pull request by shakirmengrani**
# Description

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by daixiang0** [comment]
@shakirmengrani thanks for your contribution! Please fix DCO first.

File: go.mod [hunk 1]
```diff
@@ -337,3 +337,5 @@ replace (
 )
 
 replace github.com/eclipse/paho.mqtt.golang => github.com/shivamkm07/paho.mqtt.golang v1.3.6-0.20220106130409-e28a1db639f8
+
+replace github.com/dapr/components-contrib => ../components-contrib
```
Comment (daixiang0) [inline go.mod:341]: We do not use this way, please implement it in components-contrib repo, after it merged, update components-contrib version here.

**Comment by yaron2** [comment]
@shakirmengrani I'm not seeing any actual code in this PR.

**Comment by daixiang0** [comment]
ping @shakirmengrani

**Pull request closed without merging**